.github/workflows/convertKqlFunctionYamlToArmTemplate.yaml

```diff
@@ -63,18 +61,23 @@ jobs:
           fi
       - name: Push changes
         uses: ad-m/github-push-action@master
-        if: ${{ env.armTemplatesChanged == 'true' }}
+        if: ${{ env.armTemplatesChanged == 'true' && !github.event.pull_request.head.repo.fork }}
         with:
             github_token: ${{ secrets.GITHUB_TOKEN }}
             repository: ${{github.event.pull_request.head.repo.full_name}}
             branch: ${{ github.head_ref }}
       - name: Add comment
         uses: mshick/add-pr-comment@v1
-        if: ${{ env.armTemplatesChanged == 'true' }}
+        if: ${{ env.armTemplatesChanged == 'true' && !github.event.pull_request.head.repo.fork }}
         with:
           message: |
             ASIM parsers have been changed. ARM templates were regenerated from the updated KQL function YAML files.
             To find the new ARM templates, pull your branch.
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           repo-token-user-login: 'github-actions[bot]' # The user.login for temporary GitHub tokens
           allow-repeats: false # This is the default
+      - name: Forked repos
+        if: ${{ env.armTemplatesChanged == 'true' && github.event.pull_request.head.repo.fork }}
+        run: |
+          echo "ASIM parsers have been changed. ARM templates should be generated from the updated KQL function YAML files. Please run the script .script\kqlFuncYaml2ArmOnForkedRepo.ps1."
```

Comment:
What  I meant here is that you will point to a file with all the commands. To help, I created it and created a short link: https://aka.ms/ASimForkedRepos that you can use here. Instead of "Please run..." add "Please refer to https://aka.ms/ASimForkedRepos for further information."